nilearn/datasets/atlas.py

```diff
@@ -327,14 +357,22 @@ def fetch_atlas_harvard_oxford(atlas_name, data_dir=None,
 
     Returns
     -------
-    data : sklearn.datasets.base.Bunch
+    data : :func:`sklearn.utils.Bunch`
         Dictionary-like object, keys are:
 
-        - "maps": nibabel.Nifti1Image, 4D maps if a probabilistic atlas is
-          requested and 3D labels if a maximum probabilistic atlas was
-          requested.
+            - 'maps': :class:`~nibabel.nifti1.Nifti1Image`, 4D maps if a
+              probabilistic atlas is requested, and 3D labels if a maximum
+              probabilistic atlas was requested.
+
+                .. note::
 
-        - "labels": string list, labels of the regions in the atlas.
+                    For some atlases, it can be the case that some regions
+                    are no longer present. In this case, the integers in the
+                    3D parcellation image might not be consecutive.
+
+            - 'labels': :obj:`list` of :obj:`str`, list of labels for the
+              regions in the atlas.
+            - 'filename': Same as 'maps'.
```

Comment:
If "maps" is a niimg instead of the path to the file, then this shouldn't be the same, right?

Reply:
Absolutely! I opened #3088 for this specific issue.